configs/TAKERG4AIO/config.h

```diff
@@ -32,6 +32,15 @@
 #define USE_FLASH
 #define USE_FLASH_W25Q128FV
 #define USE_MAX7456
+// Added to enable barometer and specify types
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_BARO_DPS310
+#define USE_BARO_MS5611
+// Added to enable magnetometer and specify types
+#define USE_MAG
+#define USE_MAG_QMC5883
+#define USE_MAG_IST8310
```

Comment:
we never have ifndef in config files, also this will gate, that mag drivers will be included because config.h comes before most of the other config files.